modal/partial_function.py

```diff
@@ -332,6 +334,7 @@ def _asgi_app(
     *,
     label: Optional[str] = None,  # Label for created endpoint. Final subdomain will be <workspace>--<label>.modal.run.
     custom_domains: Optional[Iterable[str]] = None,  # Deploy this endpoint on a custom domain.
+    requires_proxy_auth: bool = False,  # Require Proxy-Authorization HTTP Headers on requests to the endpoint
```

Comment:
Nit, but is "the endpoint" ambiguous for the asgi_app/wsgi_app/web_server decorators?